docs/en/reference/advanced-configuration.rst

```diff
@@ -290,7 +290,7 @@ instance of ``Doctrine\DBAL\Connection``. If an array is passed it
 is directly passed along to the DBAL Factory
 ``Doctrine\DBAL\DriverManager::getConnection()``. The DBAL
 configuration is explained in the
-`DBAL section <./../../../../../dbal/latest/docs/reference/configuration/en>`_.
+`DBAL section <./../../../../../projects/doctrine-dbal/en/latest/reference/configuration.html>`_.
```

Comment:
I suggest using an absolute link rather than hacking a relative link, which would break when reorganizing the ORM doc too